.gitignore

```diff
@@ -1,7 +1,56 @@
-.DS_Store
-node_modules
+# build
+bundle.js
+
```

Comment:
So just have dist in .gitignore but not .npmignore right? Since we want to publish dist.